<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit8088"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ip</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditip"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditread">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit_4"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonsend">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>362</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonclose">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="3">
    <widget class="QPushButton" name="pushButtonconnect">
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
